dags/multipod/maxtext_nightly.py

```diff
@@ -33,12 +33,49 @@
     catchup=False,
 ) as dag:
   # Maxtext
-  maxtext_nightly_v4_8 = maxtext_gce_config.get_maxtext_nightly_config(
+  maxtext_nightly_1slice_v4_8 = maxtext_gce_config.get_maxtext_nightly_config(
       tpu_version=TpuVersion.V4,
       tpu_cores=8,
       tpu_zone=Zone.US_CENTRAL2_B.value,
       time_out_in_min=60,
+      is_tpu_reserved=False,
+      test_name="maxtext-nightly",
```

Comment:
Nit: has a variable for this `maxtext-nightly` and used for each test?

Reply:
Done.